expression-syntax.md

```diff
@@ -1,68 +1,36 @@
 ---
 title: Expression Syntax
 summary: Learn about the expression syntax in TiDB.
-aliases: ['/docs/dev/expression-syntax/','/docs/dev/reference/sql/language-structure/expression-syntax/']
```

Comment:
The two aliases should not be deleted. Please add them back. Also, [the alias in the Chinese version](https://github.com/pingcap/docs-cn/pull/3313/files#diff-79967a03c6e0a734c3eff924e7a60662L4) need to be added back. Could you help open a PR in docs-cn repo to add it back? @ireneontheway

Reply:
ok